crates/virtio-bindings/src/virtio_mmio.rs

```diff
@@ -0,0 +1,31 @@
+/* automatically generated by rust-bindgen */
```

Comment:
Since we now have these defined here, should we actually use them in the virtio-device implementation instead of having magic offsets: https://github.com/rust-vmm/vm-virtio/blob/main/crates/virtio-device/src/mmio.rs#L72?

We will need to disambiguate between the offsets (which are defined in this file) and the actual values, defined in mmio.rs. It's a bit of bummer that these defines don't have `OFFSET` in the name :( but nothing we can do about that.

Reply:
> Since we now have these defined here, should we actually use them in the virtio-device implementation instead of having magic offsets: https://github.com/rust-vmm/vm-virtio/blob/main/crates/virtio-device/src/mmio.rs#L72?

Makes sense for sure. Should I do it in this series itself or do it once this is merged ?

> We will need to disambiguate between the offsets (which are defined in this file) and the actual values, defined in mmio.rs.

Well the offsets are prefixed with VIRTIO_, which the values aren't. Anything else we want to update in mmio.rs.

Reply:
Ah, I missed that, then we don't need to update anything else. I would say to add the change in the same PR because this is the PR that adds the values which make the change needed.

Reply:
Done.